src/scipy_api_list.py

```diff
@@ -56,11 +56,15 @@ def generate_scipy_apis(pymod) -> dict:
     d = defaultdict(list)
     attributes = sorted(set(pymod.__all__))
     for attr in attributes:
+        is_const = True
         for (key, method) in types:
             condition = eval(f"inspect.{method}(pymod.{attr})")
             if condition:
                 d[key].append(attr)
+                is_const = False
                 break
+        if is_const:
+            d["constant"].append(attr)
```

Comment:
[Question] Why this handling is needed? the constant module is different from other modules?

Reply:
Let's say we want to generate API to get mathematical/physical constants e.g. `scipy.constants.golden_ratio` and `scipy.constants.gram`. They are defined in `scipy/scipy/constants/_constants.py`:

https://github.com/scipy/scipy/blob/74946d2b20d03af9c5b6caa0d34b8c76a52333c8/scipy/constants/_constants.py#L68-L76

https://github.com/scipy/scipy/blob/74946d2b20d03af9c5b6caa0d34b8c76a52333c8/scipy/constants/_constants.py#L122

We note that they are defined in the form of `<variable_name> = <value>`, which is not treated as a `function`, `module` nor `class`, which means the Python module `inspection` can't capture them.

Therefore to collect value of constants, I have to update `generate_scipy_apis` at least.